assets/src/web-stories-block/css/lightbox.css

```diff
@@ -0,0 +1,78 @@
+/* Lightbox styles */
+.web-stories-list .web-stories-list__lightbox {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  background: black;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity 0.2s linear;
+}
+
+.web-stories-list .web-stories-list__lightbox.show {
+  opacity: 1;
+  visibility: visible;
+  z-index: 99999;
+}
+
+.web-stories-list .web-stories-list__lightbox amp-story-player {
+  width: 100%;
+  height: 100%;
+}
+
+.web-stories-list .story-lightbox__close-button {
+  display: inline-block;
+  position: fixed;
+  left: 0;
+  top: 5px;
+  cursor: pointer;
+  height: 40px;
+  width: 40px;
+  z-index: 1;
+}
+
+.web-stories-list .story-lightbox__close-button--stick {
+  display: inline-block;
+  position: absolute;
+  left: 20px;
+  top: 15px;
+  height: 20px;
+  width: 3px;
+  background-color: #fff;
+  transform: rotateZ(45deg);
+}
+
+.web-stories-list .story-lightbox__close-button--stick:first-child {
+  transform: rotateZ(-45deg);
+}
+
+/*
+ * amp-story-player's 'shadow-root-intermediary' element shows black space on top,
+ * because of the 'a' links. Making 'a' absolutely positioned removes them from normal
+ * relative flow, removes the space.
+ */
+html:not([amp])
+  .web-stories-list
+  .web-stories-list__lightbox
+  amp-story-player
+  a {
+  position: absolute;
+}
+
+@media all and (min-width: 676px) {
```

Comment:
I didn't know you couldnt use css variables in media queries. https://www.w3.org/TR/css-variables-1/#using-variables